dummy/neovim/__init__.py

```diff
@@ -0,0 +1,11 @@
+"""Python client for Nvim.
+
+Client library for talking with Nvim processes via it's msgpack-rpc API.
```

Comment:
"its"

Reply:
Should probably change this line anyway.